graphics/vid.cpp

```diff
@@ -417,11 +417,11 @@ namespace Vid
       width  = cr.Width();
       height = cr.Height();
 
-      S32 l = (CurMode().rect.Width()  - width)  >> 1;
-      S32 t = (CurMode().rect.Height() - height) >> 1;
-			SetWindowPos( hWnd, HWND_TOP, l, t, width, height, NULL);
-
-		  SetRects();
+      S32 l = (GetSystemMetrics(SM_CXSCREEN) - width)  >> 1;
```

Comment:
This won't work for a multi-monitor setup, DR2 allows to choose which monitor you want the game to run on. Maybe its best to leave as-is or rewrite and test for multi monitor setups (so it centers on the selected screen by the user).